samples/smoketest/event_hubs_async.py

```diff
@@ -15,7 +15,7 @@ def __init__(self):
         connection_string = os.environ["EVENT_HUBS_CONNECTION_STRING"]
         event_hub_name = "myeventhub"
         self.client = EventHubClient.from_connection_string(
-            connection_string, event_hub_name
+            connection_string, event_hub_path=event_hub_name
```

Comment:
This is for an API Change in eventhub right? (the change from positional to keyword arg)

Reply:
It is :)